pkgs/racket-doc/scribblings/reference/stx-trans.scrbl

```diff
@@ -224,6 +224,46 @@ an identifier, the @racket[exn:fail:contract] exception is raised.
   (slv-lookup slv-inst-2)
 ]
 
+In the above example, the struct is @racket[free-identifier=?] to the renaming
+target identifier.
+If, however, this is not desired, such as if the struct is mutable, this can be
+prevented by setting the @racket['not-free-identifier=?]
+@racket[syntax-property] to @racket[#t] will prevent this.
```

Comment:
I like the idea of linking to `make-rename-transformer`, in fact, we might be able to do it without an example.